tplus/client/orderbook.py

```diff
@@ -323,6 +348,37 @@ async def get_user_orders_for_book(
         parsed_orders = parse_orders(response_data)
         return parsed_orders, response_data
 
+    @staticmethod
```

Comment:
IMO `@staticmethod` is typically unnecessary, might as well use a module level method or a `@classmethod`, depending. Like I think type checkers also get made when you call a static method from an instance

Reply:
good call out, moved it to a module-level function